Userland/Libraries/LibJS/Runtime/VM.cpp

```diff
@@ -298,17 +296,15 @@ ThrowCompletionOr<void> VM::property_binding_initialization(BindingPattern const
             continue;
         }
 
-        Optional<Reference> reference_to_assign_to;
-
-        property.alias.visit(
-            [&](Empty) {},
-            [&](NonnullRefPtr<Identifier> const& identifier) {
-                reference_to_assign_to = resolve_binding(identifier->string(), environment);
+        Optional<Reference> reference_to_assign_to = TRY(property.alias.visit(
```

Comment:
can we use `auto` here?

Reply:
Made this one and a similar one auto. Bit sad that the right hand side has to mention the type 4 times but :shrug: